client/src/components/MultiJurisdictionAudit/RoundManagement/BatchRoundDataEntry.test.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { screen, fireEvent } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { useParams } from 'react-router-dom'
+import BatchRoundDataEntry from './BatchRoundDataEntry'
+import { roundMocks, contestMocks } from '../useSetupMenuItems/_mocks'
+import { batchesMocks, batchResultsMocks, INullResultValues } from './_mocks'
+import { renderWithRouter, withMockFetch } from '../../testUtilities'
+import { IContest } from '../../../types'
+import { IBatch } from './useBatchResults'
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'), // use actual for all non-hook parts
+  useRouteMatch: jest.fn(),
+  useParams: jest.fn(),
+}))
+const paramsMock = useParams as jest.Mock
+paramsMock.mockReturnValue({
+  electionId: '1',
+  jurisdictionId: '1',
+})
```

Comment:
do these all need to be mocked? i would have thought they would work when using `renderWithRouter`

Reply:
@jonahkagan I tried not mocking them but it broke. :shrug:

Reply:
dang

Reply:
don't necessarily need to dig into this too far, but DataEntry/index.test.tsx seems to not have this issue. maybe worth checking out

Reply:
@jonahkagan :monocle_face: